Mage.Sets/src/mage/cards/g/GandalfWhiteRider.java

```diff
@@ -0,0 +1,135 @@
+package mage.cards.g;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.SpellCastControllerTriggeredAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.BoostControlledEffect;
+import mage.abilities.effects.keyword.ScryEffect;
+import mage.cards.Card;
+import mage.constants.*;
+import mage.abilities.keyword.VigilanceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.events.ZoneChangeEvent;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+
+/**
+ *
+ * @author TiagoMDG
+ */
+public final class GandalfWhiteRider extends CardImpl {
+
+    public GandalfWhiteRider(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{3}{W}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.AVATAR);
+        this.subtype.add(SubType.WIZARD);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Vigilance
+        this.addAbility(VigilanceAbility.getInstance());
+
+        // Whenever you cast a spell, creatures you control get +1/+0 until end of turn. Scry 1.
+        // When Gandalf, White Rider dies, you may put it into its owner's library fifth from the top.
+
+        // Adds ability that whenever you cast a spell it gives a temporary +1/+0 effect until end of turn
+        Ability ability = new SpellCastControllerTriggeredAbility(
+                new BoostControlledEffect(1, 0, Duration.EndOfTurn).setText("creatures you control get +1/+0 until end of turn"),
```

Comment:
nit, is there a reason this text needs to be set manually? seems straightforward enough that auto generation should be correct